dust-tt/dust
# Add: support for multi regions in auth flow

## Description

- Redirect the user to the right region after auth0 flow.
- Removed resolver middle man.

## Risk

Low, behind dev flag.

To test locally:
- pick a region name N that starts with "local" and a port P
- add the entry to the list REGIONS_TO_URLS
- restart the regular front server
- start another front server on port P and override the DUST_REGION env var with N (DUST_REGION=N npm run...)
- As both will answer that they know the user (same DB), go to the second one in the list, sign in and you should be redirected to the first one.

## Deploy Plan

Deploy `front`
